libraries/c_sdk/aws/defender/test/system/aws_iot_tests_defender_system.c

```diff
@@ -18,83 +18,63 @@
  * COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER
  * IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
  * CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
- *
- * http://aws.amazon.com/freertos
- * http://www.FreeRTOS.org
```

Comment:
I don't believe these lines should've been deleted

Reply:
See 709cbc3ad